### DISCUSSION TITLE
Right alignment to the buttons of tabview

### DISCUSSION THREAD

**Discussion by FarhanRah** [TomSchimansky/CustomTkinter#1282] [Q&A]
Is it possible to do right alignment to the buttons of tabview? By default they seem to be aligned to the center of the widget, but I want to align it to the right side. This is my code:
```
import customtkinter

app = customtkinter.CTk()

app.columnconfigure(0, weight=1)
app.rowconfigure(0, weight=1)

notebook = customtkinter.CTkTabview(app)

my_font = customtkinter.CTkFont(size=20) #Font obeject

notebook.add("Tab1")
notebook.add("Tab2")
notebook.grid(padx=10, pady=10, sticky="news")
for button in notebook._segmented_button._buttons_dict.values():
    button.configure(width=100, height=50, font=my_font) #Change font using font object
app.mainloop()
```

**Comment by ElectricCandlelight** [comment]
Same answer as #1104 

``tabview._segmented_button.grid(sticky="E")
``
where tabview is the name of your tabview object.

**Comment by MaxNiark** [comment]
Have fun evevryone : .place and .grid method 

![image](https://user-images.githubusercontent.com/126392328/223542785-65b86222-341c-400b-b13e-4266b07a32c3.png)

**Reply by MaxNiark** [reply]
x +=50 

![image](https://user-images.githubusercontent.com/126392328/223542944-cc524093-959b-40b8-87e9-5038b5bef975.png)

**Comment by ElectricCandlelight** [comment]
If posting code please use code blocks when sharing code. <CTRL + E> or use the <> in the menu.

If posting multiple lines start with three backticks  `` ```Python New line then close with another three backticks``` ``

Plus I would avoid using place unless you really have to and use pack() or grid() instead.

**Reply by MaxNiark** [reply]
you are right ! i share the code ( with . gird method() )

i get another issue now. 

there are 3 items : 
 -CTkSegmentedButton
 -CTkFrame 
 -CTkCanvas

i need to set the place of CTkFrame because it's in front of CTkSegmentedButton in CTkCanvas .. ^^ 

do you know how can i set the grid of CTkCanvas for CTkFrame widget ? 

Always more question than solution. 

```
import customtkinter

customtkinter.set_appearance_mode("dark")  # Modes: "System" (standard), "Dark", "Light"
customtkinter.set_default_color_theme("green")  # Themes: "blue" (standard), "green", "dark-blue"

Tab_name="bouh"


class MyTabView(customtkinter.CTkTabview):
    def __init__(self, master, **kwargs):
        super().__init__(master, **kwargs)

        self.add(Tab_name)
        self.add("bouh2")


        # add widgets
        
        self.label = customtkinter.CTkLabel(master=self.tab("bouh2"))
        self.label.place(x=30, y=30)


class App(customtkinter.CTk):
    def __init__(self):
        super().__init__()

        self.title("CustomTkinter complex_example.py")
        self.geometry(f"{1100}x{580}")
        tab_view = MyTabView(master=self, width=1250, height=800)
        tab_view.grid(sticky="e")
        print("_segmented_button",tab_view._segmented_button._buttons_dict.values())
        print(tab_view._tab_dict.values())


        tab_view._segmented_button.configure(height=150,width=50, bg_color="blue")
        
        i=0
        for buttons in tab_view._segmented_button._buttons_dict.values():
            buttons.configure(height=20, width=30)     
            buttons.grid(row=i, column=0)   
            i+=1
        tab_view._segmented_button.grid(sticky="w")
        

app = App()
app.mainloop()
```
